backend/corpora/parliament/citation/netherlands.md

```diff
@@ -0,0 +1,33 @@
+## Citing the entire corpus
+
+People & Parliament presents the *Dutch parliamentary data* corpus, which is a combination of the following:
+- Dutch parliamentary proceedings from 1814-2013, harvested and enriched in the [Political Mashup project](https://ssh.datastations.nl/dataset.xhtml?persistentId=doi:10.17026/dans-xk5-dw3s), retrieved 2020
+- Dutch parliamentary proceedings from 2014-2022, harvested and enriched by [ParlaMINT](https://www.clarin.eu/parlamint), first retrieved 2020 and updated 2023
+
+### APA style
+
+> University of Jyväskylä and Utrecht University (2020, 2023). *Dutch Parliamentary data* [data set]. People & Parliament: the Netherlands. {{ frontend_url }}/search/parliament-netherlands
+
+### MLA style
+
+[MLA guidelines](https://style.mla.org/) recommend against citing a database, and recommend [citing each individual work you use](https://style.mla.org/separate-entries-database-works/). If you want to cite the entire corpus nonetheless, we recommend the following format:
+
+> University of Jyväskylä and Utrecht University. "Dutch Parliamentary data". *People & Parliament*, 2020/2023. {{ frontend_url }}/search/parliament-netherlands
+
+## Referring to a debate
+To get an URL for an entire debate, you can use the *view debate* link for a speech. This will get you a link like this:
+
+    {{ frontend_url }}/search/parliament-netherlands?debate_id=ParlaMint-NL_2021-12-21-eerstekamer-4&sort=sequence,asc
+
+## Citing a specific speech
+
+To cite a speech in the *Dutch Parliamentary data* corpus, you can retrieve a link by clicking the *link* icon underneath the speech's document tile. This should give you an url as follows:
+{{ frontend_url }}/document/parliament-netherlands/ParlaMint-NL_2021-12-21-eerstekamer-4.u1
+
+### APA style
+
+> Rutte, M. (2021). In *Report of the meeting of the Dutch Lower House, Meeting 37, Session 2 (2021-12-21)*. {{ frontend_url }}/document/parliament-netherlands/ParlaMint-NL_2021-12-21-tweedekamer-2.u225
+
+### MLA style
+
+> Rutte, Mark. *Report of the meeting of the Dutch Lower House, Meeting 37, Session 2 (2021-12-21)*, 2021. {{ frontend_url }}/document/parliament-netherlands/ParlaMint-NL_2021-12-21-tweedekamer-2.u225
```

Comment:
I remember from my time on the project that the "official" English translations for eerste kamer / tweede kamer are house of commons / senate. Did you get this format from somewhere?

Reply:
This description comes from the actual data (debate title).